config/runtime.exs

```diff
@@ -11,62 +11,62 @@ alias Jellyfish.ConfigReader
 config :ex_dtls, impl: :nif
 config :opentelemetry, traces_exporter: :none
 
-hosts = ConfigReader.read_nodes("NODES")
+nodes = ConfigReader.read_nodes("JF_NODES")
 
-if hosts do
+if nodes do
   config :libcluster,
     topologies: [
       epmd_cluster: [
         strategy: Cluster.Strategy.Epmd,
-        config: [hosts: hosts]
+        config: [hosts: nodes]
       ]
     ]
 end
 
 prod? = config_env() == :prod
 
-host =
-  case System.get_env("HOST") do
-    nil when prod? -> raise "Unset HOST environment variable"
-    nil -> "localhost"
-    other -> other
-  end
+ip =
+  ConfigReader.read_ip("JF_IP") ||
+    Application.get_env(:jellyfish, JellyfishWeb.Endpoint)[:http][:ip]
 
 port =
-  ConfigReader.read_port("PORT") ||
+  ConfigReader.read_port("JF_PORT") ||
     Application.get_env(:jellyfish, JellyfishWeb.Endpoint)[:http][:port]
 
+host =
+  case System.get_env("JF_HOST") do
+    nil -> :inet.ntoa(ip) |> to_string()
+    other -> other
+  end
+
 config :jellyfish,
-  webrtc_used: ConfigReader.read_boolean("WEBRTC_USED") || true,
-  integrated_turn_ip: ConfigReader.read_ip("INTEGRATED_TURN_IP") || {127, 0, 0, 1},
-  integrated_turn_listen_ip: ConfigReader.read_ip("INTEGRATED_TURN_LISTEN_IP") || {127, 0, 0, 1},
+  webrtc_used: ConfigReader.read_boolean("JF_WEBRTC_USED") || true,
+  integrated_turn_ip: ConfigReader.read_ip("JF_INTEGRATED_TURN_IP") || {127, 0, 0, 1},
+  integrated_turn_listen_ip:
+    ConfigReader.read_ip("JF_INTEGRATED_TURN_LISTEN_IP") || {127, 0, 0, 1},
   integrated_turn_port_range:
-    ConfigReader.read_port_range("INTEGRATED_TURN_PORT_RANGE") || {50_000, 59_999},
-  integrated_turn_tcp_port: ConfigReader.read_port("INTEGRATED_TURN_TCP_PORT"),
+    ConfigReader.read_port_range("JF_INTEGRATED_TURN_PORT_RANGE") || {50_000, 59_999},
+  integrated_turn_tcp_port: ConfigReader.read_port("JF_INTEGRATED_TURN_TCP_PORT"),
   jwt_max_age: 24 * 3600,
-  output_base_path: System.get_env("OUTPUT_BASE_PATH", "jellyfish_output") |> Path.expand(),
-  address: System.get_env("JELLYFISH_ADDRESS") || "#{host}:#{port}",
-  metrics_ip: ConfigReader.read_ip("METRICS_IP") || {127, 0, 0, 1},
-  metrics_port: ConfigReader.read_port("METRICS_PORT") || 9568
+  output_base_path: System.get_env("JF_OUTPUT_BASE_PATH", "jellyfish_output") |> Path.expand(),
+  address: "#{host}:#{port}",
```

Comment:
Why did you remove `JELLYFISH_ADDRESS` variable?

Reply:
As in the PR description :stuck_out_tongue:

Reply:
Are you sure that this won't break when you use for example proxy? Besides IMO it will look strange if somebody set `JF_HOST` to some domain and then jellyfish will send his clients to connect to `domain`:`PORT`.

Reply:
Why? #{host}:#{port} was default in the previous implementation. That's also our HTTP server address and when we open WS connection we can use FQDN, we don't have to use ip address. We also don't use this for WebRTC stuff so I would try with this setup as it limits the number of env vars.

Reply:
After discussing the problem offline, I added JF_HOST_PORT.

The problem was that without JF_HOST_PORT, we would return invalid port to Jellyfish Client .e.g 5002 instead of 443 (when running behind proxy) so we either have to allow to pass port in JF_HOST or add additional env var. I opt for additional env var as it is easier to understand and describe those 4 variables (JF_HOST, JF_HOST_PORT, JF_IP, JF_PORT). See .env.sample for more detailed explanation.

cc @sgfn @roznawsk